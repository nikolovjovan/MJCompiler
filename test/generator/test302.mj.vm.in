-5
8
29