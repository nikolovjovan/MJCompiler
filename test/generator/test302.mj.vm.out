0165
-4
32-5829
58
Completion took 0 ms